pkgs/applications/networking/cluster/argocd-cli/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, buildGoPackage, fetchFromGitHub, packr }:
+
+buildGoPackage rec {
+  pname = "argocd-cli";
+  version = "1.3.0";
```

Comment:
Latest version is https://github.com/argoproj/argo-cd/releases/tag/v1.4.2

Reply:
Updated

Reply:
done